docs/developer/guide.md

```diff
@@ -31,3 +31,9 @@ The following steps are needed to introduce a new resource and its respective re
 | EXPERIMENTAL           | basemetrics.ALPHA  |
 | STABLE                 | basemetrics.STABLE |
 
+### Common test failures
+
+1. File is not `goimports`-ed with -local k8s.io/kube-state-metrics (goimports)
+```
+make lint-fix
+```
```

Comment:
This seems self-explanatory IIUC, because on such an error `golangci-lint` would point the user to exactly where (and why) it's coming from. Do we want to add this section?

Reply:
I agree

Reply:
Thanks! Removed.